src/main/java/httpServer/ListenerAndResponder.java

```diff
@@ -90,7 +92,7 @@ private HTTPRequestBuilder readHeaders(ClientReadable bufferedReader, HTTPReques
         String[] metadata = headers.toString().split(CRLF);
 
         for (String metadataLine: metadata) {
-            requestBuilder = isRequestLine(metadataLine) ? requestBuilder.buildRequestLine(metadataLine) : requestBuilder.buildHeaderLine(metadataLine);
+            requestBuilder = requestParserHelper.isRequestLine(metadataLine) ? requestBuilder.buildRequestLine(metadataLine) : requestBuilder.buildHeaderLine(metadataLine);
```

Comment:
I think I mentioned in the last PR, but it's a little confusing to call the `StringBuilder` instance `headers` and this method itself is more than just reading `headers`.

Reply:
Changed it to reflect that it's both status code and headers!